src/layman/layer/client_test.py

```diff
@@ -90,24 +88,9 @@ def test_post_layers_chunk(chrome):
     button = button[0]
     button.click()
 
-    time.sleep(0.5)
-
-    with app.app_context():
-        layer_url = url_for('rest_workspace_layer.get', workspace=WORKSPACE, layername=LAYERNAME)
-    response = requests.get(layer_url)
-    keys_to_check = ['db_table', 'wms', 'wfs', 'thumbnail', 'file', 'metadata']
-    max_attempts = 20
-    attempts = 1
-    while not (response.status_code == 200 and all(
-            'status' not in response.json()[k] for k in keys_to_check
-    )):
-        # print('waiting')
-        time.sleep(0.5)
-        response = requests.get(layer_url)
-        attempts += 1
-        if attempts > max_attempts:
-            chrome.save_screenshot('/code/tmp/artifacts/client-post-layers-2.5.png')
-            raise Exception('Max attempts reached!')
+    process_client.wait_for_publication_status(WORKSPACE,
```

Comment:
It can be useful to have client's thumbnail saved in case of exception. I suggest something like
```
try:
    process_client.wait_for_publication_status(WORKSPACE,
                                                   process_client.LAYER_TYPE,
                                                   LAYERNAME)
except Exception as e:
    chrome.save_screenshot('/code/tmp/artifacts/client-post-layers-2.5.png')
    raise e
```